README.md

```diff
@@ -23,36 +50,79 @@ that's been made default from that version forward.
 
 ## Installation
 
-Webpacker is currently compatible with Rails 4.2+, but there's no guarantee it will still be
-in the future.
+Webpacker is currently compatible with Rails 4.2+. You can either make use of Webpacker during setup of a new Rails 5.1+ application with `--webpack` option.
 
-You can either make use of Webpacker during setup of a new application with `--webpack`
-or you can add the gem and run `./bin/rails webpacker:install` in an existing application.
+```bash
+# Available Rails 5.1+
+rails new webpacker-app --webpack
+```
 
-As the rubygems version isn't promised to be kept up to date until the release of Rails 5.1, you may want to include the gem directly from GitHub:
+Or, you can add the gem to your `Gemfile`, run bundle and `./bin/rails webpacker:install` in an existing Rails application.
 
 ```ruby
-gem 'webpacker', github: 'rails/webpacker'
+# Gemfile
+gem 'webpacker', github: '~> 1.2'
+```
+
+## Integrations
+
+Webpacker also ships with basic out-of-the-box integration for React, Angular, Vue and Elm and you can add them in your Rails app like so,
+
+```bash
+# For Rails 5.1+
+rails new webpacker-example-app --webpack=react
+rails new webpacker-example-app --webpack=angular
+rails new webpacker-example-app --webpack=vue
+```
+
+Or, by running installer manually once you have
+bundled the `webpacker` gem into your `Gemfile`
+
+```bash
+# Available integrations - [react, angular, vue or elm]
+rails webpacker:install:react
 ```
 
-You can also see a list of available commands by running `./bin/rails webpacker`
+You can see a list of available commands/tasks by running `./bin/rails webpacker`
 
 ## Binstubs
 
 Webpacker ships with two binstubs: `./bin/webpack` and `./bin/webpack-dev-server`.
-They're thin wrappers around the standard webpack.js executable, just to ensure that the right configuration file is loaded depending on your environment.
+They're thin wrappers around the standard webpack.js executable,
+just to ensure that the right configuration file is loaded depending on your environment.
 
-In development, you'll need to run `./bin/webpack-dev-server` in a separate terminal from `./bin/rails server` to have your `app/javascript/packs/*.js` files compiled as you make changes. If you'd rather not have to run the two processes separately by hand, you can use [Foreman](https://ddollar.github.io/foreman). `./bin/webpack-dev-server` launches the [Webpack Dev Server](https://webpack.js.org/configuration/dev-server/), which serves your pack files on http://localhost:8080/, and provides advanced Webpack features, such as [Hot Module Replacement](https://webpack.js.org/guides/hmr-react/).
+You can also pass cli options supported by [webpack-dev-server](https://webpack.js.org/configuration/dev-server/). Please note that inline
+options will always take precedence over the ones
+already set in the configuration file.
+
+```bash
+./bin/webpack-dev-server --host 0.0.0.0
```

Comment:
I didn't realize we allowed/encouraged passing options like `--host` and probably broke it in https://github.com/rails/webpacker/commit/2a94cdf1c93e0509af0cce971a788e5eea6ca8e3 because it constructs `ASSET_HOST` [using `development.server.yml` exclusively](https://github.com/rails/webpacker/blob/2a94cdf1c93e0509af0cce971a788e5eea6ca8e3/lib/install/bin/webpack-dev-server.tt#L27-L28). `bin/webpack-dev-server` will need updating.